examples/n-body/assembly/index.ts

```diff
@@ -5,7 +5,7 @@ import "allocator/arena";
 
 type float = f64; // interchangeable f32/f64 for testing
 
-const SOLAR_MASS = <float>(4.0 * Math.PI * Math.PI);
+const SOLAR_MASS: float = 4.0 * Math.PI * Math.PI;
```

Comment:
I think this won't work if `type float = f32` because `Math.PI` is `f64`.

Reply:
Hmm, I thought `const a: type = b` and `const a = <type>(b)` has same result. Isn't true?

Reply:
Exceptions from this rule are `NaN` and `Infinity` that derive their type (either f32 or f64) from context atomatically.

Reply:
Got it. That's good to know

Reply:
Fixed!